src/app/code/community/Zendesk/Zendesk/controllers/Adminhtml/ZendeskController.php

```diff
@@ -364,9 +364,10 @@ public function autocompleteAction()
     public function logAction()
     {
         $path = Mage::helper('zendesk/log')->getLogPath();
-        
+
         if(!file_exists($path)) {
             Mage::getSingleton('adminhtml/session')->addError(Mage::helper('zendesk')->__('The Zendesk log file has not been created. Check to see if logging has been enabled.'));
+            file_put_contents(" ",$path);
```

Comment:
What was creating the log in the past?
Shouldn't the log message above be changed to "Creating a Zendesk log file"?